packages/next-templates/src/app/klachtenformulier-niet-ingelogd/stap2/page.tsx

```diff
@@ -0,0 +1,210 @@
+    }
+  }, []);
+
+  return (
+    <Page>
+      <ExampleHeader />
+      <ExampleNavigation />
+      <PageContent className="voorbeeld-page-content-flex">
+        <BreadcrumbNav className="voorbeeld-breadcrumb-nav" label="Kruimelpad">
+          <BreadcrumbNavLink href="/klachtenformulier-niet-ingelogd/stap1">
+            <IconArrowLeft></IconArrowLeft>Vorige Stap
+          </BreadcrumbNavLink>
+        </BreadcrumbNav>
+
+        <Article id="main" className="voorbeeld-article-space voorbeeld-main-layout">
+          <div>
+            <Heading1>Klacht over de gemeente doorgeven </Heading1>
+            <Paragraph lead>Stap 2 van 4</Paragraph>
+            <Paragraph>Vul de vakjes met een * altijd in. Anders kunt u niet verder.</Paragraph>
+
+            <Heading2>Uw gegevens</Heading2>
+
+            <form>
+              <Fieldset>
+                <FieldsetLegend>Bent u een persoon of een bedrijf?</FieldsetLegend>
+                <FormField type="radio">
+                  <Paragraph className="utrecht-form-field__label utrecht-form-field__label--radio">
+                    <FormLabel htmlFor="persoon" type="radio">
+                      <RadioButton
+                        name="persoon-of-bedrijf"
+                        id="persoon"
+                        className="utrecht-form-field__input"
+                      ></RadioButton>
+                      Persoon
+                    </FormLabel>
+                  </Paragraph>
+                </FormField>
+
+                <FormField type="radio">
+                  <Paragraph className="utrecht-form-field__label utrecht-form-field__label--radio">
+                    <FormLabel htmlFor="bedrijf" type="radio">
+                      <RadioButton
+                        name="persoon-of-bedrijf"
+                        id="bedrijf"
+                        className="utrecht-form-field__input"
+                      ></RadioButton>
+                      Bedrijf
+                    </FormLabel>
+                  </Paragraph>
+                </FormField>
+              </Fieldset>
+
+              {/* staat in figma niet in fieldset - wel gedaan want h3 per sectie in form - h3 in fieldsetlegend */}
+              <Fieldset>
+                <FieldsetLegend>
+                  <Heading3>Persoonsgegevens</Heading3>
+                </FieldsetLegend>
+                <FormField>
+                  <FormLabel htmlFor="voorletter">Voorletter(s)</FormLabel>
+                  <FormFieldDescription></FormFieldDescription>
+                  {/* zonder formfielddescription komt textbox naast formlabel te staan, met descrpt. eronder?  */}
+                  <Textbox id="voorletter"></Textbox>
+                  {/* is in Figma een korte textbox, bestaat die optie? */}
+                </FormField>
+
+                <FormField>
+                  <FormLabel htmlFor="tussenvoegsel">Tussenvoegsel(s)</FormLabel>
+                  <FormFieldDescription></FormFieldDescription>
```

Comment:
Wordt hier later nog iets aan toegevoegd?

Reply:
Als ik die weghaal, komt de text area naast het label te staan.. Ik snapte niet waarom dus heb die vraag er in een comment bijgezet maar de opbouw voor nu wel zo gelaten omdat het visueel staat zoals het hoort.

Reply:
Bedoel je dat het een sort flexbox geeft

Reply:
Dat is prima als je het zo laat maar dan misschien een issue aanmaken dat dit visueel goed is en moet nog gefikst worden :)

Reply:
Ik heb hier een issue van gemaakt op de backlog. :)